# -*- coding: utf-8 -*-
# Generated by Django 1.9.1 on 2016-01-29 21:01
from __future__ import unicode_literals

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='CharityCheck',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('datetime_checked', models.DateTimeField(auto_now=True)),
                ('ein', models.CharField(max_length=100, unique=True)),
                ('name', models.CharField(max_length=100)),
                ('city', models.CharField(max_length=100)),
                ('state', models.CharField(max_length=2)),
                ('country', models.CharField(max_length=50)),
                ('deductability_code', models.CharField(max_length=100)),
                ('verified', models.BooleanField(default=False)),
                ('sanitized', models.BooleanField(default=False)),
                ('charity', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='charity_check', to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.CreateModel(
            name='CharityCheckException',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('message', models.CharField(max_length=200)),
                ('datetime_checked', models.DateTimeField(auto_now_add=True)),
                ('charity_check', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='exceptions', to='charity_check.CharityCheck')),
            ],
        ),
    ]
